content/manuals/docker-hub/repos/manage/trusted-content/dvp-program.md

```diff
@@ -14,91 +14,171 @@ aliases:
+  with Docker Scout
+- [Priority search ranking](#priority-search-ranking): Enhanced discoverability
+  in Docker Hub search results
+- [Removal of rate limiting](#removal-of-rate-limiting): Unrestricted pulls for
+  development teams
+- [Co-marketing opportunities](#co-marketing-opportunities): Joint promotional
+  activities with Docker
+
+### Enterprise-grade infrastructure
+
+The Docker Verified Publisher Program runs on Docker Hub's enterprise-scale
+infrastructure, serving millions of developers globally. Your published content
+benefits from:
+
+- High availability and uptime: Docker's systems are designed for failover
+  across multiple availability zones, with load-balanced autoscaling, enabling
+  99.9% uptime.
+- Global delivery and fast downloads: Docker leverages Cloudflare's CDN and
+  caching (with Cache Reserve) to achieve cache hit ratios more than 99%,
+  reducing reliance on origin traffic and ensuring fast access for developers
+  everywhere.
+- Durability: Docker maintains a documented backup policy and performs full
+  daily backups of production data.
+
+You simply push your images to Docker Hub as usual, and Docker takes care of the
+rest, serving your image to millions of developers worldwide.
+
+![DVP flow in Docker Hub](./images/dvp-hub-flow.svg)
+
+To learn more, see [Availability at
+Docker](https://www.docker.com/trust/availability/).
 
-The Docker Verified Publisher Program (DVP) provides several features and benefits to Docker
-Hub publishers. The program grants the following perks based on participation tier:
+### Verified publisher badge
 
-- Repository logo
-- Verified publisher badge
-- Priority search ranking in Docker Hub
-- Insights and analytics
-- Vulnerability analysis
-- Additional Docker Business seats
-- Removal of rate limiting for developers
-- Co-marketing opportunities
+Images that are part of this program have a special badge on Docker Hub making
+it easier for users to identify projects that Docker has verified as
+high-quality commercial publishers.
 
-### Repository logo
+![Docker-Sponsored Open Source
+badge](../../../images/verified-publisher-badge.png)
 
-DVP organizations can upload custom images for individual repositories on Docker Hub.
-This lets you override the default organization-level logo on a per-repository basis.
+### Repository logo
 
-Only a user with administrative access (owner or team member with administrator permission)
-over the repository can change the repository logo.
+DVP organizations can upload custom images for individual repositories on Docker
+Hub. This lets you override the default organization-level logo on a
+per-repository basis.
 
-#### Image requirements
+Only a user with administrative access (owner or team member with administrator
+permission) over the repository can change the repository logo. To manage the
+repository logo, see [Manage repository logo](#manage-repository-logo).
```

Comment:
q: Does this mean a user with owner/editor role? If so let's maybe clarify that here

https://docs.docker.com/enterprise/security/roles-and-permissions/#organization-roles